src/RaisedButton/RaisedButton.js

```diff
@@ -61,6 +61,7 @@ function getStyles(props, context, state) {
     root: {
       display: 'inline-block',
       transition: transitions.easeOut(),
+      width: fullWidth ? '100%' : 'auto',
```

Comment:
It's also possible to fix this by changing `display` to `block` instead of `inline-block` when `fullWidth` is set, instead of adding `width`. Not sure what is better.